### ISSUE TITLE
Sizes of tensors must match except in dimension 1. Expected size 11 but got size 21 for tensor number 1 in the list.

### ISSUE THREAD

**Issue by celinecsw** [comfyanonymous/ComfyUI#8793]
### Custom Node Testing

- [x] I have tried disabling custom nodes and the issue persists (see [how to disable custom nodes](https://docs.comfy.org/troubleshooting/custom-node-issues#step-1%3A-test-with-all-custom-nodes-disabled) if you need help)

### Expected Behavior

Error occurred when executing undefined:  Sizes of tensors must match except in dimension 1. Expected size 11 but got size 21 for tensor number 1 in the list.  File "/workspace/ComfyUI/execution.py", line 1060, in execute output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb) File "/workspace/ComfyUI/execution.py", line 275, in get_output_data return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb) File "/workspace/ComfyUI/custom_nodes/ComfyUI-0246/utils.py", line 353, in new_func res_value = old_func(*final_args, **kwargs) File "/workspace/ComfyUI/execution.py", line 247, in _map_node_over_list process_inputs(input_dict, i) File "/workspace/ComfyUI/execution.py", line 236, in process_inputs results.append(getattr(obj, func)(**inputs)) File "/workspace/ComfyUI/custom_nodes/ComfyUI-WanVideoWrapper/nodes.py", line 3820, in process noise_pred_context, new_teacache = predict_with_cfg( File "/workspace/ComfyUI/custom_nodes/ComfyUI-WanVideoWrapper/nodes.py", line 3523, in predict_with_cfg [noise_pred_cond, noise_pred_uncond], cache_state_cond = transformer( File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl return self._call_impl(*args, **kwargs) File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1747, in _call_impl return forward_call(*args, **kwargs) File "/workspace/ComfyUI/custom_nodes/ComfyUI-WanVideoWrapper/wanvideo/modules/model.py", line 1295, in forward render_latent = torch.cat([hidden_states[:, :20], pcd_data["render_latent"]], dim=1) 

Close
Idle

### Actual Behavior

Error occurred when executing undefined:  Sizes of tensors must match except in dimension 1. Expected size 11 but got size 21 for tensor number 1 in the list.  File "/workspace/ComfyUI/execution.py", line 1060, in execute output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb) File "/workspace/ComfyUI/execution.py", line 275, in get_output_data return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb) File "/workspace/ComfyUI/custom_nodes/ComfyUI-0246/utils.py", line 353, in new_func res_value = old_func(*final_args, **kwargs) File "/workspace/ComfyUI/execution.py", line 247, in _map_node_over_list process_inputs(input_dict, i) File "/workspace/ComfyUI/execution.py", line 236, in process_inputs results.append(getattr(obj, func)(**inputs)) File "/workspace/ComfyUI/custom_nodes/ComfyUI-WanVideoWrapper/nodes.py", line 3820, in process noise_pred_context, new_teacache = predict_with_cfg( File "/workspace/ComfyUI/custom_nodes/ComfyUI-WanVideoWrapper/nodes.py", line 3523, in predict_with_cfg [noise_pred_cond, noise_pred_uncond], cache_state_cond = transformer( File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl return self._call_impl(*args, **kwargs) File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1747, in _call_impl return forward_call(*args, **kwargs) File "/workspace/ComfyUI/custom_nodes/ComfyUI-WanVideoWrapper/wanvideo/modules/model.py", line 1295, in forward render_latent = torch.cat([hidden_states[:, :20], pcd_data["render_latent"]], dim=1) 

Close
Idle

![Image](https://github.com/user-attachments/assets/d4e291ee-933b-47cd-90d2-bd1c48f951a7)

### Steps to Reproduce

https://www.runninghub.cn/workflow/1940976653524955137?source=workspace

### Debug Logs

```powershell
Error occurred when executing undefined:  Sizes of tensors must match except in dimension 1. Expected size 11 but got size 21 for tensor number 1 in the list.  File "/workspace/ComfyUI/execution.py", line 1060, in execute output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb) File "/workspace/ComfyUI/execution.py", line 275, in get_output_data return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb) File "/workspace/ComfyUI/custom_nodes/ComfyUI-0246/utils.py", line 353, in new_func res_value = old_func(*final_args, **kwargs) File "/workspace/ComfyUI/execution.py", line 247, in _map_node_over_list process_inputs(input_dict, i) File "/workspace/ComfyUI/execution.py", line 236, in process_inputs results.append(getattr(obj, func)(**inputs)) File "/workspace/ComfyUI/custom_nodes/ComfyUI-WanVideoWrapper/nodes.py", line 3820, in process noise_pred_context, new_teacache = predict_with_cfg( File "/workspace/ComfyUI/custom_nodes/ComfyUI-WanVideoWrapper/nodes.py", line 3523, in predict_with_cfg [noise_pred_cond, noise_pred_uncond], cache_state_cond = transformer( File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl return self._call_impl(*args, **kwargs) File "/usr/local/lib/python3.10/dist-packages/torch/nn/modules/module.py", line 1747, in _call_impl return forward_call(*args, **kwargs) File "/workspace/ComfyUI/custom_nodes/ComfyUI-WanVideoWrapper/wanvideo/modules/model.py", line 1295, in forward render_latent = torch.cat([hidden_states[:, :20], pcd_data["render_latent"]], dim=1)
```

**Comment by neo-111** [comment]
Just got the same error when running a WAN2.1 workflow. It was working perfectly before. Just stopped right after 12 succesful generations, and started giving me this error.

**Comment by christian-byrne** [comment]
Which model are you using exactly? Where did you download it from?

**Comment by dsphotoblog** [comment]
same here using a multitalk workflow with wan video
